### PR TITLE
Make conditions for create / edit object more exclusive

### PR DESCRIPTION
**Pull request by Beacze**
@solth Could you review it?

### CODE REVIEW

**Comment by Beacze** [comment]
Should I use id everywhere?

**Comment by solth** [comment]
> Should I use id everywhere?

If the goal of the PR is what its title states and if it still works, than yes.

File: Kitodo/src/main/webapp/pages/templateEdit.xhtml [hunk 1]
```diff
@@ -28,9 +28,12 @@
             window.onload = function () {checkForm('editForm')};
         </script>
         <h3>
-            <h:outputText value="#{msgs.editTemplate} (#{TemplateForm.template.title})" rendered="#{not empty TemplateForm.template.title}"/>
-            <h:outputText value="#{msgs.newTemplate}" rendered="#{empty TemplateForm.template.title}"/>
-            <h:outputText value="#{msgs.duplicateTemplate}" rendered="#{empty TemplateForm.template.title and not empty TemplateForm.template.docket}"/>
+            <h:outputText value="#{msgs.editTemplate} (#{TemplateForm.template.title})"
+                          rendered="#{TemplateForm.template.id ne null and TemplateForm.template.title ne null and not empty TemplateForm.template.docket}"/>
+            <h:outputText value="#{msgs.newTemplate}"
```
Comment (solth) [inline Kitodo/src/main/webapp/pages/templateEdit.xhtml]: The conditions are now mutually exclusive, but neither "New template" nor "Duplicate template" is displayed (only "Edit template"), so apparently neither condition is met in the corresonding situation (before these changes, at least "New template" was properly displayed when creating a new template)

Comment (Beacze) [reply Kitodo/src/main/webapp/pages/templateEdit.xhtml]: So for now I will just leave it without full exclusions before it gets totally destroyed ;)

**Pull request closed without merging**